tests/schemata/datasets.yaml

```diff
@@ -59,3 +59,24 @@ datasets:
           description: Tokens in the testing subset
           format: txt
           path: wikitext-103/wiki.test.tokens
+  noaa_jfk:
```

Comment:
In the production we should have the remote default ready. But this file is for tests, and when we test functionalities we always use the test files, for reliability and decoupling development. `load_schemata` defaults to the default of each, but in test we always point them to local ones.

In fact I intentionally kept the remote default as the old one, so that we can discover testing issues (we shouldn't rely on the remote files in testing).

Reply:
Makes sense, I agree that's a good idea when developing. When do we want to update the remote one, then? After this PR is closed?

Reply:
I don't think we should update the remote url until it's sometime close to official release, at which time we should decide where to host these files and the actual content of these files. I believe we have contained testing on these remote default files to the XFAIL one.

Reply:
Ok but `list_all_datasets` won't work properly and `load_dataset` won't be able to find `jfk_noaa` either. As long as we're ok with that for demos. It also makes it a bit annoying to test loading a new dataset, but I guess it's not too big a deal:

```python
import pydax.dataset
import os
import pathlib
dataset_url = pathlib.Path(os.getcwd()) / 'tests/schemata/datasets.yaml'
schemata = pydax.load_schemata(dataset_url=dataset_url)
jfk_schema = schemata.schemata['dataset_schema'].export_schema('datasets', 'noaa_jfk', '1.1.4')
data_dir = pydax.get_config().DATADIR / 'jfk' / '1.1.4'
d = pydax.dataset.Dataset(schema=jfk_schema, data_dir=data_dir, mode=pydax.dataset.Dataset.InitializationMode.DOWNLOAD_AND_LOAD)
d.data['jfk_weather_cleaned'].head()
```

Reply:
I see what you meant. OK, let me update the current remote default

Reply:
updated

Reply:
thanks